ledger-service/http-json/README.md

```diff
@@ -71,7 +71,7 @@ supports such tokens.
 ### POST http://localhost:7575/contracts/search
 application/json body:
 ```
-{"templateIds": [{"moduleName": "Iou", "entityName": "Iou"}]}
+{"%template": [{"moduleName": "Iou", "entityName": "Iou"}]}
```

Comment:
what singular template instead of plural templates?
and what is the meaning of the percent sign?